src/js/templates/application/student/singleApplicationTemplate.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Creates the HTML for a single application
+ * @param {object} data - The single application data fetched from the API
+ * @returns - The HTML for a single application
+ */
+export function singleApplicationTemplate() {
+  /// Placeholders. Needs to be swapped with data parameter
+  const data = {
+    listingTitle: 'Listing title',
+    media: 'https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/Placeholder_view_vector.svg/1362px-Placeholder_view_vector.svg.png?20220519031949',
+    jobTitle: 'Job title',
+    companyName: 'Company name',
+    location: 'Location',
+    listingCreated: 'One day ago',
+    deadline: 'Deadline date',
+    applicantsCount: '48 applicants',
+    applicantsName: 'Applicants name',
+    applicationCreated: 'Time / date',
+    email: 'Email',
+    address: 'Address',
+    phone: 'Phone number',
+    applicationText: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum enim facilisis gravida neque convallis a cras. Ut faucibus pulvinar elementum integer enim neque.',
+    link: 'Link (Link added by student',
+    file: 'File (file added by student)',
+  };
+
+  const { media, listingTitle, jobTitle, companyName, location, listingCreated, deadline, applicantsCount, applicantsName, applicationCreated, email, address, phone, applicationText, link, file, id } = data;
+
+  /// Element
```

Comment:
@asimenstad 
I would create a factory function to create all of these. It will make the code more DRY. 

ex: 
```js
 export function displayFactory(elementName = '', className = [], elementId = '', ElementInner = ``) {
    const element = document.createElement(elementName);
    element.classList.add(...className);
    element.id = elementId;
    element.innerHTML = ElementInner;
    return element;
  }

const displayUserImage = displayFactory('div', ['user-image', 'd-block'], 'userImage', `<img src="${imageUrl}" class="ag-profile-img shadow ag-relative" />`);
```

Reply:
If you are unsure on how to use this I can explain a it more. Just let me know :;)